uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/controllers/main.js

```diff
@@ -10,15 +10,27 @@ define([
     function ($scope, $rootScope, $rootElement, $log, $modal, $q, FileService, config) {
       $log.debug('Controller: MainCtrl');
 
-      $rootScope.modalDocument = function (data, e) {
+      /**
+       * Opens Document Modal based on passed document data
```

Comment:
It also check the mode of the modal, please try to document the function so that a layman can understand what it does.